<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>history_batch_set_dialog</class>
 <widget class="QDialog" name="history_batch_set_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>查询时间段</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="start_lb">
        <property name="text">
         <string>开始时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="CDateEdit" name="start_dedt" native="true">
        <property name="styleSheet">
         <string notr="true">QDateEdit
{
    border: 1px solid #dbdbdb;
    color: #666666;
    font: 30px &quot;微软雅黑&quot;;
}

QDateEdit:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}

QDateEdit::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 40px;

    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}

QDateEdit::down-arrow {
    image: url(:/icon/icon/calendar.ico);
}

QDateEdit::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="stop_lb">
        <property name="text">
         <string>结束时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="CDateEdit" name="stop_dedt" native="true">
        <property name="styleSheet">
         <string notr="true">QDateEdit
{
    border: 1px solid #dbdbdb;
    color: #666666;
    font: 30px &quot;微软雅黑&quot;;
}

QDateEdit:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}

QDateEdit::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 40px;

    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}

QDateEdit::down-arrow {
    image: url(:/icon/icon/calendar.ico);
}

QDateEdit::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>256</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="select_pb">
        <property name="text">
         <string>开始查询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cur_batch_lb">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTableWidget" name="batch_tablewidget"/>
      </item>
      <item>
       <widget class="QTableWidget" name="mis_tablewidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>305</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="ok_pb">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cache_pb">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QWidget</extends>
   <header location="global">open_lib/cdateedit/cdateedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
